sdk/python/feast/errors.py

```diff
@@ -78,6 +78,16 @@ def __init__(self, offline_store_name: str, data_source_name: str):
         )
 
 
+class FeatureNameCollisionError(Exception):
```

Comment:
It's not clear to me why we are passing in refs here when we don't use the feature view name. What if we had something like 

```
def __init__(self, feature_views: List[str], feature_name: str):
```
That would be easier for me to understand as well. The current argument is cryptic.